include/internal/bopomofo-private.h

```diff
@@ -37,6 +37,7 @@ enum {
     KB_DVORAK,
     KB_DVORAK_HSU,
     KB_DACHEN_CP26,
+    KB_CARPALX,
```

Comment:
Please put `KB_CARPALX` just above `KB_TYPE_NUM` so that the index of original keyboard layouts will not changed.